docs/reference/instructor_reports.rst

```diff
@@ -0,0 +1,6 @@
+.. _instructor-reports:
```

Comment:
I think this one must be with underscore, same for `operator-reports`

Reply:
Yeah, latest commit has the fixes 🤞  also a make target for checking this locally since I always have some warning that blows it up.